### DISCUSSION TITLE
Can I make a release on my fork?

### DISCUSSION THREAD

**Discussion by rognoni** [chris1111/Wireless-USB-Big-Sur-Adapter#181] [Ideas]
Hello,
thank you for your job 👍
 Can I make a release on [my fork](https://github.com/rognoni/Wireless-USB-Big-Sur-Adapter) to **backup** your useful work? (adding your ZIP file `Wireless.USB.Big.Sur.Adapter-V12.zip`)

**Comment by chris1111** [comment]
Yes you can
